src/index.html

```diff
@@ -11,7 +11,146 @@
+          max="100">
+        </div>
+        <div class="form-field">
+          <label for="birth">Full date of birth:</label>
+          <input id="birth"
+          type="date"
+          name="date of birth">
+        </div>
+        <div class="form-field">
+          <label for="terms">I accept the term of the agreement</label>
+          <input id="terms"
+          type="checkbox"
+          name="terms"
+          required>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+          <input id="email"
+          type="email"
+          name="email"
+          placeholder="email@example.com">
+        </div>
+        <div class="form-field">
+          <label for="password">Password:</label>
+          <input type="password"
+          name="password"
+          id="password"
+          minlength="3"
+          maxlength="15">
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label for="yes">
+            <input type="radio"
+            name="yes"
+            id="yes">
+            Yes
+          </label>
+          <label for="no">
+            <input type="radio"
+            name="no"
+            id="no">
+            No
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="color">
+            What is your favorite color?
+            <input type="color"
+            name="color"
+            id="color">
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="bedTime">
+            What time do you go to bed?
+            <input type="time"
+            id="bedTime"
+            name="bed-time">
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="car">
+            What are your favorite brand of cars?
+            <select id="car"
+            multiple>
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+              <option value=""></option>
+            </select>
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="rate">
+            How do you rate our worl?
+            <input type="range"
+            id="rate"
+            value="0">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Additional info:</legend>
+        <div class="form-field">
+          <label for="comments">
+            Comments:
+            <textarea name=""
+            id="comments">
+            </textarea>
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="final-question">
+            Would you recommend us?
+            <select id="final-question">
```

Comment:
Use camelCase for Id
```suggestion
            <select id="finalQuestion">
```